hawc/apps/materialized/management/commands/refresh_materialized_views.py

```diff
@@ -0,0 +1,28 @@
+from django.apps import apps
```

Comment:
I'd just make them create all and refresh all mvs - if we're calling this for one I think we're calling for all of them.  I think I'd just remove the optional argument to simplify.

Reply:
The only problem with combining create / refresh is that indices are also made on create, so subsequent runs of this combined command will duplicate the indices.